frontend/src/components/Dropdown/components/MultiSelectCombobox/MultiSelectCombobox.tsx

```diff
@@ -25,7 +25,8 @@ const MultiItemsContainer: FC = ({ children }) => {
       // Margin difference for selected items.
       my="-3px"
       // Padding for dropdown toggle.
-      maxW="calc(100% - 2.5rem)"
+      flexBasis="calc(100% - 2.5rem)"
+      minW="0px"
```

Comment:
We want to try and avoid modifying the components. Is there a solution where we just modify the layout higher up in the tree? Perhaps we can pass in some `flex` props in the `EditConditionBlock` to make it work.

Reply:
This addition is a static addition that is harmless in normal use (correct?), I'm thinking it's acceptable 🤔

Reply:
Do correct me if I'm wrong, but I think this line change is unnecessary based on my observations on the UI behaviour.